cmds/find/find.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/u-root/u-root/pkg/find"
+)
+
+var (
+	perm   = flag.Int("mode", -1, "Permissions")
+	ftype  = flag.String("type", "", "File type")
+	name   = flag.String("name", ".*", "re for name")
```

Comment:
I did a little discussion on the unix history list about this. The name glob came from an earlier kernel (i.e. pre-unix) and from what people said the glob syntax preceded regular expression usage in Unix. We don't use REs today in some places because glob came first. It's history, not a conscious decision. I decided on this pass -- why not try REs and see if we like them?

Reply:
btw find was not viewed with favor by its creators and was in fact removed from Plan 9. So I figure a new start with the same name is forgiveable.

Reply:
I think this would be too finicky and surprising considering that a lot of filename contain a '.' and that '.' has special meaning in regex. Consider this find:

    find --name ha.a

Would also match "haha".

Or, if you consider finding files which begin with a period:

    find --name .*
    find --name ..*
    find --name \..*

None of these would work. In the last one, most shells would convert '\.' to '.' before sending as argv. These two would work:

    find --name \\..*
    find --name '..*'

I believe findutils has it right. Use the `-regex` flag for regular expressions and the `-name` flag for non-regex matching. You can still decide not to support globs in `-name`.

Having both commands is trival to add with this function: https://godoc.org/regexp#QuoteMeta